filebeat/module/haproxy/log/test/tcplog.log-expected.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "@timestamp": "2018-09-20T15:44:23.285Z", 
+        "fileset.module": "haproxy", 
+        "fileset.name": "log", 
+        "haproxy.backend_name": "app", 
+        "haproxy.backend_queue": 0, 
+        "haproxy.bytes_read": 212, 
+        "haproxy.client_ip": "127.0.0.1", 
+        "haproxy.client_port": 40962, 
+        "haproxy.connection_wait_time_ms": -1, 
+        "haproxy.connections.active": 1, 
+        "haproxy.connections.backend": 0, 
+        "haproxy.connections.frontend": 1, 
+        "haproxy.connections.retries": 0, 
+        "haproxy.connections.server": 0, 
+        "haproxy.default.logsource": "127.0.0.1", 
+        "haproxy.frontend_name": "main", 
+        "haproxy.pid": 25457, 
+        "haproxy.process_name": "haproxy", 
+        "haproxy.server_name": "<NOSRV>", 
```

Comment:
It may worth to handle `<NOSRV>` and leave it empty in the event. You could also add an example that fills this value with a real server name.

Reply:
I'm not sure how to achieve this. I don't see any kind of conditional post-processor to remove a field if it matches some pattern. What am I missing?

Reply:
Oh ok, I see, leave it then.